push/entrypoint.sh

```diff
@@ -34,3 +30,10 @@ if [ $LATEST = true ]; then
     echo "Pushing $GCLOUD_REGISTRY/$IMAGE:latest"
     docker push $GCLOUD_REGISTRY/$IMAGE:latest
 fi
+
+for tag in $TAGS
+do
+  echo "Pushing $GCLOUD_REGISTRY/$IMAGE:$tag"
+  docker tag $GCLOUD_REGISTRY/$IMAGE:$TAG $GCLOUD_REGISTRY/$IMAGE:$tag
```

Comment:
What is `$TAG` here?
```suggestion
  docker tag $GCLOUD_REGISTRY/$IMAGE:$TAG $GCLOUD_REGISTRY/$IMAGE:$tag
```

Reply:
Should maybe be hardcoded `$GITHUB_SHA` or configurable as `$SRC_TAG`.

Reply:
The thing is that this tag is something that needs to be there when the push action runs.